test/Test/Indigo/Contracts/HomebaseLite/Metadata/Views.hs

```diff
@@ -35,19 +33,12 @@ test_proposalInfo =
     let uri = #proposal_uri :! URI "ipfs://proposal"
         choices = ["Zero", "One", "Two", "Three"]
     withSender holder do
-      call contract (Call @"Propose") (uri, #choices :! choices)
+      transfer contract $ calling (ep @"Propose") (uri, #choices :! choices)
     storage <- getStorage contract
     let proposals = sProposalsRPC storage
     proposal <- getBigMapValue proposals uri
-    Showing <$> callOffChainView contract "proposalInfo" (ViewParam uri)
-      @@== Showing proposal
-    res <- attempt $
+    Showing <$> callOffChainView @ProposalInfo contract "proposalInfo" (ViewParam uri)
+      @@== Showing (toRPCProposalInfo proposal)
+    expectTransferFailure (failedWith (constant @MText "NoSuchProposal")) $
```

Comment:
Uh. I think this is the same as

```suggestion
    expectCustomErrorNoArg #noSuchProposal
```

When I said "use `expectTransferFailure`" I didn't mean that quite so literally :sweat_smile: sorry if I wasn't clear.